site/content/gitops-quickstart/01-apply-gitops.md

```diff
@@ -0,0 +1,260 @@
+    --git-email happy@gopher.org \
+    --name wonderful-wardrobe-1565767990
+[...]
+[ℹ]  Flux will only operate properly once it has write-access to the Git repository
+[ℹ]  please configure git@github.com:happy-gopher/flux-get-started so that the following Flux SSH public key has write access to it
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCt8no0/F3+kD1YukH6sVIv1ONcy9+01G2/AQe1CQA+uRHaioep41U3ghROU7CoM1yTjG+eLYgu26UMvkXVbOmEm+1697adh4qz/yCF0E7JtCIIXGn/1XrLb6OxgtlGKdJ4fTUdxtQSyTvWqjxQhC4ute9hnHWU8oiSrNaq5D20P5x8sgPf4V0A5YWD5S4YliJcIupTzrD7zjhh6TyP5fqhPLHPBZFHStHq0DSD+Gi6vXZz1s9UmuAnxP8pkIlrW22xJyFbsmcjJuks5FvmLo8uJMeWTx5t+3WKWp8ZKrbDJFUWQ8aVMByHYq1c3doevM28CHwz/
+```
+
+Copy the lines starting with `ssh-rsa` and add it as a deploy key, to
+e.g. Github. There you can easily do this in the
+`Settings > Deploy keys > Add deploy key`. Just make sure you check
+`Allow write access` as well.
+
+The next time Flux syncs from git, it will start updating the cluster
+and actively deploying. If you use the [`flux-get-started`
+repo](https://github.com/fluxcd/flux-get-started) from above, here's
+what you will see in your cluster:
+
+```console
+$ kubectl get pods -n demo
+NAME                       READY   STATUS    RESTARTS   AGE
+podinfo-5f4bd464b4-4vf7k   1/1     Running   0          58m
+podinfo-5f4bd464b4-hkgzt   1/1     Running   0          58m
+```
+
+Remember that you can further tweak the installation of Flux as
+discussed in our [`install flux` docs](/usage/experimental/gitops-flux/).
+
+### Handcrafting your configuration
+
+In this second step we will use `eksctl generate config`, so you can
+easily handcraft your cluster configuration and maintain it in Git, just
+like you would for the workloads you are running.
+
+During the previous command (`eksctl install flux`), we instructed Flux
+to watch a config repository. This is where the workloads are defined.
+Now we will add the cluster config as well.
+
+In this example we will run:
+
+```console
+EKSCTL_EXPERIMENTAL=true eksctl generate profile \
+        --name wonderful-wardrobe-1565767990 \
+        --git-url git@github.com:happy-gopher/eks-gitops-example.git \
+        --profile-path ~/dev/flux-get-started/cluster-config
+```
+
+Let's break this down here. `eksctl generate profile` at the very
+least wants:
+
+- `--name`: the name of the cluster - check `eksctl get cluster`
+  to see what the name of yours is
+- `--git-url`: the Git URL of the base config of your cluster
+- `--profile-path`: a local path: this is an empty new directory
+  (here `cluster-config`) you create in your local checkout of
+  the config repository, which we used in the previous command
+
+The base config can be something you and your organisation
+tailored to your needs, but can be something like our [EKS
+GitOps example][eks-gitops-example] as well. It is meant to be
+a starting point for clusters you can iterate over.
+
+[eks-gitops-example]: https://github.com/weaveworks/eks-gitops-example
```

Comment:
This should have been mentioned at the beginning of the page, so that people can choose whether they want to follow it?